### PR TITLE
Add Project ERD chart 

### PR DESCRIPTION
**Pull request by Georjane**
The following models were represented with their associations on an Entity Relational Diagram
- User
- Post
- Comment
- Like
- Friendship

### CODE REVIEW

File: docs/erd-document.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Entity Relationship of the ROR-SOCIAL-SCAFFOLD project
+![](./ERD.png)
```
Comment (lekegitrepo) [inline docs/erd-document.md:2]: - **_The comment should contain user_id, not author_id_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi @Georjane and @descholar-ceo :clap: :clap:
#### You have done a good job, however, there is a minor fix you need to implement before this project can be approved. Check my comment :point_down:

#### _Reviewed by Temitayo Adeleke._

**Comment by descholar-ceo** [comment]
> ## Status: Request changes 
> ### Hi @Georjane and @descholar-ceo  
> #### You have done a good job, however, there is a minor fix you need to implement before this project can be approved. Check my comment 
> #### _Reviewed by Temitayo Adeleke._

Thank you so much @lekegitrepo for this kind helpful feedback, we will implement it and re-request for new review

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Hi @Georjane and @descholar-ceo :clap: :clap:
#### Well done :muscle: you have done a good job by implementing the required changes.
#### This project is approved :white_check_mark: 

#### _Reviewed by Temitayo Adeleke._

**Pull request merged**